specification/azuredatatransfer/AzureDataTransfer.Management/Pipeline.tsp

```diff
@@ -54,50 +57,49 @@ interface Pipelines {
   >;
 
   /**
-   * Deletes the pipeline resource.
+   * Deletes the specified Pipeline resource.
    */
   delete is ArmResourceDeleteWithoutOkAsync<Pipeline>;
 
   /**
-   * Gets pipelines in a resource group.
+   * Lists all Pipeline resources within the specified resource group.
    */
   listByResourceGroup is ArmResourceListByParent<Pipeline>;
 
   /**
-   * Gets pipelines in a subscription.
+   * Lists all Pipeline resources within the current subscription.
    */
   listBySubscription is ArmListBySubscription<Pipeline>;
 
   /**
-   * Approves the specified connection request in a pipeline.
+   * Approves a pending connection request associated with the specified Pipeline.
    */
   approveConnection is ArmResourceActionAsync<
     Pipeline,
     ResourceBody,
-    ArmConnectionResponse
+    ArmResponse<Connection>
   >;
 
   /**
-   * Executes a privileged action for a pipeline.
+   * Executes a privileged or administrative action on the specified Pipeline.
    */
   executeAction is ArmResourceActionAsync<
     Pipeline,
     Action,
-    ArmPipelineResponse
+    ArmResponse<Pipeline>
```

Comment:
In general, this should just be the body type (in this case Pipeline)  in every case.